framework/src/main/java/org/apache/drill/test/framework/ConnectionPool.java

```diff
@@ -27,27 +27,20 @@
 import java.util.Map;
 import java.util.Queue;
 
-public class ConnectionPool implements AutoCloseable {
+public class ConnectionPool implements DrillDefaults, AutoCloseable {
   private static final Logger LOG = Logger.getLogger(ConnectionPool.class);
-  private String URL_STRING = "jdbc:drill:drillbit=localhost";
-  private String JDBC_DRIVER = "org.apache.drill.jdbc.Driver";
+  //private String URL_STRING = "jdbc:drill:drillbit=localhost";
+  //private String JDBC_DRIVER = "org.apache.drill.jdbc.Driver";
   //private static final String URL_STRING = Utils.getDrillTestProperties().get("CONNECTION_STRING");
   //private static final String URL_STRING = String.format("jdbc:drill:drillbit=%s",
   //  Utils.getDrillTestProperties().get("DRILL_STORAGE_PLUGIN_SERVER"));
 
```

Comment:
wanted to store it somewhere in case we need to look it up for the drivers and connection string.